linkcheck/htmlutil/linkparse.py

```diff
@@ -175,10 +175,11 @@ def start_element (self, tag, attrs, element_text, lineno, column):
         log.debug(LOG_CHECK, "line %d col %d", lineno, column)
         if tag == "base" and not self.base_ref:
             self.base_ref = attrs.get("href", u'')
-        if tag =="a" and attrs.get_true('class', u''):
-           if ("broken_link" in attrs.get('class')):
-               log.debug(LOG_CHECK," Found a broken_link to %s, not considering further", attrs.get('href'))
-               return
+        ignoreclasses = ['broken_link','borked_link']
```

Comment:
I suggest passing ignoreclasses (maybe better spelled `ignore_classes` for readability?) to the `__init__` method and stashing them as an attribute on self.

There's only one place (outside test code) where a LinkFinder is created: find_links() in `linkcheck/parser/__init__.py`.  It can access the config via `url_data`, which is an instance of UrlBase.  There's precedent in linkcheck/plugins/sslcertcheck.py:

```python
    config = url_data.aggregate.config
```

so you can then do

```python
    handler = linkparse.LinkFinder(callback, tags, ignore_class=config['ignoreclass'])
```